docs/controllers/services/examples/README.md

```diff
@@ -154,5 +157,30 @@ spec:
         ports:
         - containerPort: 80
           protocol: TCP
+```
+
+## Surfacing errors related to provisioning a load balancer
+
+Cloud Controller Manager is using DigitalOcean API internally to provision a
+DigitalOcean load balancer. It might happen that provisioning will be
+unsuccessful, because of various reasons.
 
+You can find provisioning status and all the reconciliation errors that Cloud
+Controller Manager encountered during the LoadBalancer service life-cycle in
+the service's event stream.
+
+Provided that your load balancer service name is `my-load-balancer`, you can use
```

Comment:
I might be biased since we're also working on that LB naming feature, but _load balancer service name_ could be confusing as to whether it refers to the LB or the service.

How about just calling it _service name_ and use the example name `my-svc`?

Reply:
@timoreimann sounds good! It is done, I just pushed the update in d420e930 :smile_cat: